apps/linode-marketplace-influxdb/roles/post/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+-
+# motd and creds gen
+- name: copy MOTD template to /etc/motd
+  template:
+    src: templates/motd.j2
+    dest: /etc/motd
+
+- name: writing influx creds into file
```

Comment:
update task to

- name: writing sudo + Influx creds into file
  copy:
    dest: '/home/{{ username }}/.credentials'
    mode: '0600'
    owner: '{{ username }}'
    group: '{{ username }}'
    content: |
      Sudo Username: {{ username }}
      Sudo Password: {{ password }}

Reply:
corrected in [0b8e27b](https://github.com/akamai-compute-marketplace/marketplace-apps/pull/120/commits/0b8e27bb9b6641c2719566c7c337299997d95b54)